src/guide/javascript.md

```diff
@@ -773,355 +439,59 @@ You can use this folder structure as a reference:
 ╰───┴──────────────────────────────╯
 ```
 
-### Client configuration
-
-Our client config is the following:
-
-```jsonc
-// FILE: config.json
-{
-  "torii": {
-    "apiURL": "http://127.0.0.1:8080",
-    "telemetryURL": "http://127.0.0.1:8081"
-  },
-  "account": {
-    "name": "alice",
-    "domain_id": {
-      "name": "wonderland"
-    }
-  },
-  "publicKey": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
-  "privateKey": {
-    "digestFunction": "ed25519",
-    "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
-  }
-}
-```
+:::code-group
```

Comment:
do you think it should be a single code-group? I'd split it in multiple ones, e.g. we can have a code group for `components/` - it makes sense, thematically. Just putting all the files for the app in a single group doesn't seem like a good choice to me.

Reply:
Actually, I don't consider the whole 8th section useful at all, because in comparison with the rest of this tutorial it has (and had) too many code with almost no detailed explanation. I am not sure, but maybe it makes sense just to leave a link to the source code so the reader might follow it and explore on their own.

Reply:
I think it's useful for some users to have this info displayed like this, and others can go look at the project sources